services/src/integ-test/resources/application-integ.yaml

```diff
@@ -43,3 +43,9 @@ sdt:
       receiveTimeout: ${RECEIVE_TIMEOUT:30000}
       idleTaskExecutionLimit:  ${IDLE_TASK_EXECUTION_LIMIT:10}
       idleConsumerLimit:  ${IDLE_CONSUMER_LIMIT:5}
+  purge:
+    cron: ${PURGE_SCHEDULE_CRON:0 10 5 * * ?}
```

Comment:
Should the default value be `0 10 0 * * ?` to match the actual correct value for the purge job schedule?

Reply:
Don't think it needs to. it's just a test.